jupyter_client/kernelspec.py

```diff
@@ -73,6 +73,21 @@ def __init__(self, name):
         self.name = name
 
 class KernelSpecManager(LoggingConfigurable):
+
+    kernel_spec_class = DottedObjectName(
```

Comment:
Okay, let me change that.  Basically just looked at MultiKernelManager for how to set that up.  Is there a better example in the codebase?